include/ocpp/v201/device_model_storage_sqlite.hpp

```diff
@@ -26,6 +27,7 @@ class DeviceModelStorageSqlite : public DeviceModelStorage {
     /// \brief Opens SQLite connection at given \p db_path
     /// \param db_path  path to database
     explicit DeviceModelStorageSqlite(const fs::path& db_path);
+    ~DeviceModelStorageSqlite();
```

Comment:
```suggestion
    ~DeviceModelStorageSqlite() = default;
```

This should work too right?